nncf/config/schemata/algo/quantization.py

```diff
@@ -0,0 +1,512 @@
+                "max_percentile":
+                    with_attributes(NUMBER,
+                                    description="For 'percentile' and 'mean_percentile' types - specify the percentile "
+                                                "of input value histograms to be set as the initial "
+                                                "value for maximum quantizer input",
+                                    default=MAX_PERCENTILE),
+            }
+        }
+    },
+    "additionalProperties": False,
+}
+PER_LAYER_RANGE_INIT_CONFIG_PROPERTIES = {
+    "type": "object",
+    "properties": {
+        **BASIC_RANGE_INIT_CONFIG_PROPERTIES["properties"],
+        **SCOPING_PROPERTIES,
+        "target_quantizer_group":
+            with_attributes(STRING, description="The target group of quantizers for which "
+                                                "specified type of range initialization will "
+                                                "be applied. By default specified type of range "
+                                                "initialization will be applied to all group of"
+                                                "quantizers. Optional.",
+                            enum=['activations', 'weights'])
+    }
+}
+RANGE_INIT_CONFIG_PROPERTIES = {
+    "initializer": {
+        "type": "object",
+        "properties": {
+            "range": {
+                "description": "This initializer performs forward runs of the model to be quantized using "
+                               "samples from a user-supplied data loader to gather activation and weight "
+                               "tensor statistics within the network and use these to set up initial range parameters "
+                               "for the quantizers. ",
+                "oneOf": [
+                    BASIC_RANGE_INIT_CONFIG_PROPERTIES,
+                    {
+                        "type": "array",
+                        "title": "per_layer_range_init_configuration",
+                        "items": PER_LAYER_RANGE_INIT_CONFIG_PROPERTIES
+                    }
+                ],
+            },
+        },
+        "additionalProperties": False,
+    },
+}
+BITWIDTH_ASSIGNMENT_MODE_SCHEMA = {
+    "type": "string",
+    "enum": ['strict', 'liberal'],
+    "default": "liberal",
+    "description": "The mode for assignment bitwidth to activation quantizers. In the 'strict' mode,"
+                   "a group of quantizers that feed their output to one and more "
+                   "same modules as input (weight quantizers count as well) will have the same bitwidth in the "
+                   "'liberal' mode allows different precisions within the group.\n"
+                   "Bitwidth is assigned based on hardware constraints. If multiple "
+                   "variants are possible, the minimal compatible bitwidth is chosen."
+}
+
+PRECISION_INIT_TYPES_VS_DESCRIPTION = {
```

Comment:
@ljaljushkin for the precision init part

Reply:
this part LGTM